pool/web/view/page/page_admin_custom_fields.ml

```diff
@@ -5,16 +5,16 @@ module Message = Pool_common.Message
 let base_path = "/admin/custom-fields"
 
 let form
-  ?custom_field
+  ?(custom_field : Custom_field.t option)
   Pool_context.{ language; csrf; _ }
-  sys_languages
+  tenant_languages
   flash_fetcher
   =
   let open Custom_field in
   let action =
     match custom_field with
     | None -> base_path
-    | Some f -> Format.asprintf "%s/%s" base_path (f.id |> Id.value)
+    | Some f -> Format.asprintf "%s/%s" base_path (f |> get_id |> Id.value)
```

Comment:
as mentioned in the entity comment. use `id` directly as function of `CustomField`

```suggestion
    | Some f -> Format.asprintf "%s/%s" base_path (f |> id |> Id.value)
```